transport/socks5/stream_dialer.go

```diff
@@ -18,10 +18,9 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"net"
+	"io"
 
 	"github.com/Jigsaw-Code/outline-sdk/transport"
-	"golang.org/x/net/proxy"
```

Comment:
run `go mod tidy` to remove the package from `go.mod`?

Reply:
Done! I forgot about it.